templates/lxc-archlinux.in

```diff
@@ -132,13 +138,23 @@ EOF
     [ ${nttys:-0} -gt 6 ] && echo \
       "You may want to modify container's /etc/securetty \
       file to allow root logins on tty7 and higher"
+    # fixes for systemd in lxc container
+    cat << EOF > ${path}/autodev
+#!/bin/bash
+cd ${LXC_ROOTFS_MOUNT}/dev
+mkdir net
+mknod net/tun c 10 200
+chmod 0666 net/tun
+EOF
+    chmod +x ${path}/autodev
     return 0
```

Comment:
Thanks! :) I'll let @stgraber decide if he's fine with placing a autodev-script in the container directory per default.